app/src/ui/history/compare.tsx

```diff
@@ -33,6 +33,8 @@ interface ICompareSidebarProps {
   readonly dispatcher: Dispatcher
   readonly currentBranch: Branch | null
   readonly sidebarHasFocusWithin: boolean
+  /**Determines whether or not to focus compare branch selector */
```

Comment:
:art: Let this comment breath a bit with some newlines and whitespace, and ensure it mentions that this is about the initial state of the branch selector.